lib/memory.fs

```diff
@@ -79,12 +81,17 @@ label .skip
     ret,
 end-local
 
+[host]
```

Comment:
Just random thought, but this pattern could maybe become something like `MACRO` / `ENDMACRO`.

Reply:
Definitely. But I think also target colon definitions will help with those cases?   We can add it for now though.